howfairis/requesting/get_from_github.py

```diff
@@ -0,0 +1,13 @@
+from .get_from_github_no_auth import get_from_github_no_auth
+from .get_from_github_with_auth import get_from_github_with_auth
+
+
+def get_from_github(url, url_type, apikeys=None):
+
+    github_apikey_is_invalid = apikeys is not None and (apikeys.get("github-key") is None or
+                                                        apikeys.get("github-user") is None)
```

Comment:
How about something like:

if apikeys.get("github-key") and apikeys.get("github-user"):